pages/linux/xeyes.md

```diff
@@ -0,0 +1,11 @@
+# xeyes
+
+> Display eyes on the screen that follow the mouese cursor
+
+- Launch xeyes on local display:
+
+`xeyes`
+
+- Launch xeyes on a remote display:
+
+`xeyes -display {{remote.host}}:0.0`
```

Comment:
Please change remote.host to remote_host. And what is the reasoning behind the `0.0` ?